ecdsa/party/implementation.go

```diff
@@ -278,12 +286,17 @@ func (signer *singleSigner) ensureStarted() *tss.Error {
 
 // Since storing to cache is done strictly when this signer had not yet received authority to sign, this
 // method will return a bool indicating whether it is allowed to sign.
-func (signer *singleSigner) attemptToCacheIfUnauthorizedToSign(message tss.ParsedMessage) (authorisedToSign bool) {
+func (signer *singleSigner) attemptToCacheIfShouldntSign(message tss.ParsedMessage) (shouldSign bool) {
 	signer.mtx.Lock()
 	defer signer.mtx.Unlock()
 
+	if signer.notInCommittee {
```

Comment:
why do you need to keep this state inside the signer? can't you have authoriseSignerToSign return it instead of storing?